plugin/src/test/java/io/jenkins/plugins/forensics/git/delta/GitDeltaCalculatorITest.java

```diff
@@ -42,8 +42,7 @@ public void shouldCreateEmptyDeltaIfCommitsAreInvalid() {
         GitDeltaCalculator deltaCalculator = createDeltaCalculator();
 
         FilteredLog log = createLog();
-        assertThat(deltaCalculator.calculateDelta("", "", log)).isEmpty();
-        assertThat(deltaCalculator.calculateDelta(mock(Run.class), mock(Run.class), log)).isEmpty();
+        assertThat(deltaCalculator.calculateDelta(mock(Run.class), mock(Run.class), "", log)).isEmpty();
```

Comment:
It might make sense to use `EMPTY_SCM_KEY` rather than `""` to make more clear why it is empty.